packages/react/src/components/List/ListItem/_list-item.scss

```diff
@@ -93,7 +93,7 @@
   color: $text-02;
   padding: $spacing-03 $spacing-05;
   align-items: center;
-  overflow-x: hidden;
```

Comment:
I see two actual code file changed with this PR. Just making sure it is intentional

Reply:
@davidicus For this one, yes, It was a bug I noticed as I was going through the stories. See next where the overflow is triggering a scroll. https://next.carbon-addons-iot-react.com/?path=/story/1-watson-iot-listitem--with-overflow-menu-row-actions